tools/modules/EnsEMBL/Web/Configuration/Tools.pm

```diff
@@ -322,18 +332,19 @@ sub populate_tree {
   if ($sd->ENSEMBL_PG_ENABLED) {
     my $pg_node = $tools_node->append($self->create_subnode('Postgap', 'Post-GWAS',
       [qw(
-        pg_input      EnsEMBL::Web::Component::Tools::Postgap::InputForm
-        pg_details    EnsEMBL::Web::Component::Tools::Postgap::TicketDetails
-        tickets       EnsEMBL::Web::Component::Tools::Postgap::TicketsList
+        pg_input        EnsEMBL::Web::Component::Tools::Postgap::InputForm
+        pg_details      EnsEMBL::Web::Component::Tools::Postgap::TicketDetails
+        tickets         EnsEMBL::Web::Component::Tools::Postgap::TicketsList
       )],
       { 'availability' => 1 }
     ));
 
     $pg_node->append($self->create_subnode('Postgap/Results', $result_cap,
       [qw(
-        details     EnsEMBL::Web::Component::Tools::Postgap::TicketDetails
-        ressummary  EnsEMBL::Web::Component::Tools::Postgap::ResultsSummary
-        results     EnsEMBL::Web::Component::Tools::Postgap::Results
+        newjobbuttontop  EnsEMBL::Web::Component::Tools::NewJobButton
```

Comment:
can you please remove it from postgap because we wont have a result page for postgap. I have just push this change.